test/index.js

```diff
@@ -578,27 +580,215 @@ describe('views()', () => {
 
 describe('Plugin', () => {
 
-    it('only registers once', async () => {
+    it('registers multiple times', async () => {
 
         const one = {
             name: 'one',
-
             register: function (server, options) {
 
-                return server.register(Vision);
+                server.register({
+                    plugin: Vision,
+                    options: {
+                        engines: { 'html': Handlebars },
+                        relativeTo: Path.join(__dirname, '/templates'),
+                        path: 'plugin'
+                    }
+                });
+
+                return server.route({
```

Comment:
I'm curious why you're using `return` here. I don't see a reference in the docs about needing to return anything from a plugin's `register` function.  (seen later in other call sites as well)